allure-commandline/src/main/java/io/qameta/allure/validator/LanguageValidator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2016-2024 Qameta Software Inc
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.qameta.allure.validator;
+
+import com.beust.jcommander.IParameterValidator;
+import com.beust.jcommander.ParameterException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author charlie (Dmitry Baev).
+ */
+public class LanguageValidator implements IParameterValidator {
+
+    private static final Set<String> SUPPORTED_LANGUAGES = Collections.unmodifiableSet(
```

Comment:
Is there a chance to make it aware of `i18n/<lang>` availability?

Reply:
We can, but it will be tricky to implement. IMHO, it is not worth the effort

Reply:
Well, I just thought that the contributors adding new languages, like in https://github.com/allure-framework/allure2/pull/2445, may be forgetting to update this thing because it's not very obvious. In any case, this can be solved with documentation, indeed.